src/btrfs/btrfs.go

```diff
@@ -655,29 +655,34 @@ func checkVersion() {
 		return
 	}
 	versionString := strings.TrimSpace(string(data))
-	version := strings.Replace(versionString, "Btrfs v", "", -1)
-	split := strings.Split(version, ".")
-	if len(split) != 2 {
+	splittedVersion := strings.Split(versionString, " v")
+	if len(splittedVersion) != 2 {
 		checkVersionErr = fmt.Errorf("unknown version string: %s", versionString)
 		return
 	}
-	major, err := strconv.ParseInt(split[0], 10, 64)
-	if err != nil {
-		checkVersionErr = err
-		return
-	}
-	if major < majorVersion {
-		checkVersionErr = fmt.Errorf("need at least btrfs version %d.%d, got %s", majorVersion, minorVersion, version)
+	split := strings.Split(splittedVersion[1], ".")
+	if len(split) < 2 {
```

Comment:
A version can be `4.1.2` for example. We're splitting on `.` so we would have a `len(split)` of 3 if I'm correct. So the only case where we want to return an error is when they're less than a major and minor version, no?